include/SDL_float.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copying and distribution of this file, with or without modification,
+ * are permitted in any medium without royalty provided this notice is
+ * preserved.  This file is offered as-is, without any warranty.
+ * Names of contributors must not be used to endorse or promote products
+ * derived from this file without specific prior written permission.
+ */
+
+#ifndef SDL_float_h_
+#define SDL_float_h_
+
+#include <float.h>
+
+#define SDL_FLT_IS_NULL(value)                              \
```

Comment:
Suggest SDL_FLOAT_IS_ZERO

Reply:
I understand that "zero" is better than "null", but I am a lot less convinced by changing "FLT" to "FLOAT". The standard C library defines `FLT_EPSILON` in `float.h` and SDL defines `SDL_FLT_EPSILON` in `SDL_stdinc.h`, so it would look strange to me not to use the same name everywhere.